c10/util/strong_type.h

```diff
@@ -52,13 +52,12 @@
 #include <fmt/format.h>
 #endif
 
-namespace strong
-{
+namespace strong {
 
-namespace impl
-{
-  template <typename T, typename ... V>
-  using WhenConstructible = std::enable_if_t<std::is_constructible<T, V...>::value>;
+namespace impl {
+template <typename T, typename... V>
```

Comment:
Why so many changes in this file? Is it covered by clang-format?

Reply:
Yes, large scale formatting.

Reply:
Then please remove it from the exclusion in clang_format .lintrunner rule: https://github.com/pytorch/pytorch/blob/0e39f4db927116b634eceb2dff886061548f8ed0/.lintrunner.toml#L75

Reply:
Wait, I think this file might be excluded because it's a vendored external library? https://github.com/rollbear/strong_type

Reply:
I think we should revert the changes here and mark it as excluded from clang-tidy.

Reply:
It should be placed in third-party.